roles/servicetelemetry/templates/manifest_alertmanager.j2

```diff
@@ -26,8 +26,8 @@ spec:
     - -upstream=http://localhost:9093/
     - -cookie-secret-file=/etc/proxy/secrets/session_secret
     - -openshift-service-account=alertmanager-stf
-    - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
-    - '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get"}}'
+    - '-openshift-sar={"namespace":"{{ ansible_operator_meta.namespace }}","resource": "alertmanager", "group":"monitoring.rhobs", "verb":"get"}'
```

Comment:
One issue here is that this is going to break customers using `observabilityStrategy: use_community` right?

Reply:
Yes, good catch thanks!